<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>StatisticsWindow</class>
 <widget class="QWidget" name="StatisticsWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="CloseButton">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>260</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Close</string>
   </property>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>271</width>
     <height>271</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_11">
      <item>
       <widget class="QLabel" name="label_21">
        <property name="text">
         <string>Body: </string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="BodyNameLabel">
        <property name="text">
         <string>TextLabel</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Maximal X</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="MaximalXLabel">
        <property name="text">
         <string>TextLabel</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Minimal X</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="MinimalXLabel">
        <property name="text">
         <string>TextLabel</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>Maximal Y</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="MaximalYLabel">
        <property name="text">
         <string>TextLabel</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_12">
      <item>
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Minimal Y</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="MinimalYLabel">
        <property name="text">
         <string>TextLabel</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <widget class="QLabel" name="label_7">
        <property name="text">
         <string>Maximal X Velocity</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="MaximalXVelLabel">
        <property name="text">
         <string>TextLabel</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_5">
      <item>
       <widget class="QLabel" name="label_9">
        <property name="text">
         <string>Minimal X Velocity</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="MinimalXVelLabel">
        <property name="text">
         <string>TextLabel</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_6">
      <item>
       <widget class="QLabel" name="label_12">
        <property name="text">
         <string>Maximal Y Velocity</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="MaximalYVelLabel">
        <property name="text">
         <string>TextLabel</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_7">
      <item>
       <widget class="QLabel" name="label_14">
        <property name="text">
         <string>Minimal Y Velocity</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="MinimalYVelLabel">
        <property name="text">
         <string>TextLabel</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_8">
      <item>
       <widget class="QLabel" name="label_15">
        <property name="text">
         <string>Maximal Velocity</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="MaximalVelLabel">
        <property name="text">
         <string>TextLabel</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_9">
      <item>
       <widget class="QLabel" name="label_17">
        <property name="text">
         <string>Minimal Velocity</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="MinimalVelLabel">
        <property name="text">
         <string>TextLabel</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_10">
      <item>
       <widget class="QLabel" name="label_19">
        <property name="text">
         <string>Lifetime (s)</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="LifetimeLabel">
        <property name="text">
         <string>TextLabel</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
